driver_cpl/cime_config/config_component.xml

```diff
@@ -3342,45 +3342,46 @@
     <valid_values></valid_values>
     <default_value>379.000</default_value>
     <values>
-      <value compset="^1850"					>284.7</value>
-      <value compset="^HIST.+BGC%BPRP"				>284.7</value>
-      <value compset="^HIST_CAM4%FCHM"				>0.000001</value>
-      <value compset="^2000"					>367.0</value>
-      <value compset="^1850.+DATM%NYF"				>379.000</value>
-      <value compset="^1850.+DATM%NYF.*_POP2%ECO"		>284.7</value>
-      <value compset="^2000.+DATM%NYF"				>379.000</value>
-      <value compset="^2000.+DATM%IAF"				>379.000</value>
-      <value compset="^HIST.+DATM"				>367.0</value>
-      <value compset="^4804.+DATM"				>367.0</value>
-      <value compset="^RCP2.+DATM"				>367.0</value>
-      <value compset="^RCP4.+DATM"				>367.0</value>
-      <value compset="^RCP6.+DATM"				>367.0</value>
-      <value compset="^RCP8.+DATM"				>367.0</value>
-      <value compset="^2000.+XATM"				>379.000</value>
-      <value compset="^HIST.+CAM"				>0.000001</value>
-      <value compset="^5505.+CAM"				>0.000001</value>
-      <value compset="^RCP2.+CAM"				>0.000001</value>
-      <value compset="^RCP4.+CAM"				>0.000001</value>
-      <value compset="^RCP6.+CAM"				>0.000001</value>
-      <value compset="^RCP8.+CAM"				>0.000001</value>
-      <value compset="^2013.+CAM"				>0.000001</value>
-      <value compset="^GEOS.+CAM"				>0.000001</value>
-      <value compset="^AMIP.+CAM"				>0.000001</value>
-      <value compset="^AR95"					>368.9</value>
-      <value compset="^AR97"					>368.9</value>
-      <value compset="^2003"					>367.0</value>
+      <value compset="^1850"                     >284.7</value>
```

Comment:
In the future please try to separate white space changes from code changes.   It makes it much easier to review a PR.